### PR TITLE
[C] Pilot Testing Revisions StSS pg.1-3

### PR DESCRIPTION
**Pull request by alexgoff**
JIRA: https://jira.lsstcorp.org/browse/EPO-5699

## What this change does ##

Update pages 1-3 of Surveying the Solar System per the pilot testing notes https://docs.google.com/document/d/1gvJN3a81tSRYP6jmXfWiYYayD2IKaiBTCZUGxXOvUnY/edit#

## Notes for reviewers ##

Verify that pages 1 through 3 of Surveying the Solar System correspond to the notes from pilot testing. 

Include an indication of how detailed a review you want on a 1-10 scale.
- 3

## Testing ##

Compare against testing feedback notes and validate the changes made.

## Checklist ##

If any of the following are true, please check them off
- [ ] This is a breaking change: changes in page data (`/src/data/pages/PAGE.json`) or scientific data (anything in `/static/`) and I will notify Product Marketing when it is in prod and user visible.
- [x] This change impacts several investigations (e.g. the change affects reuseed styles, widgets, pages, QAs, utility methods, etc.)
- [x] This change is isolated to a specific page or Component (i.e. it's a one-off)
- [ ] I don't know what this change does

### CODE REVIEW

File: src/assets/stylesheets/components/site/_modals.scss [hunk 1]
```diff
@@ -50,6 +50,7 @@
       padding: $minPadding;
       border-bottom-right-radius: $minPadding;
       border-bottom-left-radius: $minPadding;
+      overflow-y: auto;
```
Comment (alexgoff) [inline src/assets/stylesheets/components/site/_modals.scss:53]: added overflow to handle content larger than the modal screen

File: src/components/page/SingleCol.jsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ class Page extends React.PureComponent {
 
     return (
       <div className={singleColGrid}>
-        {reference && <Reference {...{ reference }} />}
+        {reference && <Reference {...{ reference, blockShared }} />}
```
Comment (alexgoff) [inline src/components/page/SingleCol.jsx:28]: pass `blockShared` to the modal so answers can be accessed by the `TableBlock`

File: src/components/reference/index.jsx [hunk 3]
```diff
@@ -18,6 +18,7 @@ function Reference({ reference }) {
 
   const getSingleImageClasses = imageBlocks => {
     if (
+      imageBlocks &&
```
Comment (alexgoff) [inline src/components/reference/index.jsx:21]: prevent `undefined` error

File: src/data/pages/detecting-objects.json [hunk 4]
```diff
@@ -59,13 +59,13 @@
...
-          "labelPost": " from the Sun, ",
+          "labelPost": " the Sun, ",
           "srLabel": "...",
           "answerAccessor": "select",
           "placeholder": "Select",
           "options": [
-            { "label": "closer", "value": "closer" },
-            { "label": "farther", "value": "farther" }
+            { "label": "closer to", "value": "closer" },
+            { "label": "farther from", "value": "farther" }
```
Comment (alexgoff) [inline src/data/pages/detecting-objects.json:68]: content revisions for pg.2

File: src/data/pages/semi-major-axis.json [hunk 5]
```diff
@@ -13,12 +13,40 @@
...
+  "content": "<p> By making a series of careful measurements, astronomers can determine the orbit for a newly discovered Solar System object. Three properties used to describe an orbit are used throughout this investigation. </p> <p> <a href='http://rubineducation.org/glossary/orbit-size' target='_blank'>Orbital size</a> is related to the orbit's <a href='http://rubineducation.org/glossary/semi-major-axis' target='_blank' >semi-major axis</a > (which is defined as half of the longest diameter of the object's orbit). It is measured in <a href='http://rubineducation.org/glossary/astronomical-unit-au' target='_blank' >astronomical units</a > (au). You can also think of the orbit's semi-major axis as its average distance from the Sun. </p>",
```
Comment (alexgoff) [inline src/data/pages/semi-major-axis.json:16]: content revisions for pg. 3

File: src/data/pages/semi-major-axis.json [hunk 6]
```diff
@@ -13,12 +13,40 @@
...
+          [{ "content": "Mars" }, { "content": "1.5" }],
+          [{ "content": "Jupiter" }, { "content": "5.2" }],
+          [{ "content": "Saturn" }, { "content": "9.5" }],
+          [{ "content": "Uranus" }, { "content": "19.2" }],
+          [{ "content": "Neptune" }, { "content": "30.1" }]
+        ],
+        "qaReview": false
+      }
+    ]
+  }
```
Comment (alexgoff) [inline src/data/pages/semi-major-axis.json:51]: modal with orbital sizes copied from glossary

File: src/data/pages/solar-system-introduction.json [hunk 7]
```diff
@@ -17,5 +17,5 @@
...
...size of the telescope’s mirrors and the sensitivity of its camera combine with the speed that the telescope surveys the night sky to help us see more objects in our Solar System than ever before. Each image taken by Rubin Observatory covers a large area of the sky, and captures light even from very faint objects, making it possible to measure the motions of millions of small Solar System objects per night. Many of these are brand new discoveries. </p> <p> In this investigation, you will explore real data from Rubin Observatory to develop a deeper understanding of how small objects are distributed throughout the Solar System. You will also examine the orbits of some newly detected Solar System objects, in order to classify them. Together, these observations will help us put together the story of how our Solar System formed, and give us insights into what we might find around other stars that have exoplanets. </p> <h3>Essential Questions</h3> <ul> <li> How do gravitational interactions affect the motion and orbital properties of small objects in the solar system? </li> <li> What can observations of small objects reveal about the formation and history of the Solar System? </li> </ul>"
```
Comment (alexgoff) [inline src/data/pages/solar-system-introduction.json:20]: re-wrote first question in this block for pg.1

File: src/fragments/reference.js [hunk 8]
```diff
@@ -20,10 +20,28 @@ export const buttonFragment = graphql`
   }
 `;
 
+export const tableFragment = graphql`
```
Comment (alexgoff) [inline src/fragments/reference.js:23]: add `Table` fragment to graphql query for `Reference`

**Review by blnkt — APPROVED** [review]

**Pull request merged**